### PR TITLE
Update EDW and Tensorization doc, and fix broken links

### LINKED ISSUE #107: Document ADT (adt.csv) and cross-reference (xref.csv) tables in Tensorization.md

### LINKED ISSUE #140: Refactor and update EDW documentation

### PR DESCRIPTION
**Pull request by raimonpv**
Closes #107 
Closes #140 

Doing this PR I discovered the following that we might want to tackle in new issues:
- Add commit hook to check for PHI in the wiki
- Tones of documentation are outdated. (Should we review each file after thesis?)
- By default add a documentation label on a new issue from the wiki

What do you think @erikr ?

### CODE REVIEW

**Comment by erikr** [comment]
Great ideas! Can you open new issues?

File: Electronic-data-warehouse.md [hunk 1]
```diff
@@ -566,13 +566,11 @@ inside the `UnitsCD` element.
...
+
+A brief description of each measurement found in the flowsheet can be found in the [EDW flowsheet reference](/docs/edw_flowsheet_reference.xlsx) xlsx file.
     
 #### Issues to consider when taking the data for the hd5 file
-Currently, the sql query that joins EDW tables to pull the flowsheet table is building the resulting table with some issues. 
-First of all, the table appears to have all the measurements from the patient, not only the ones during the time period related to
-the encounter ID (CSN). Therefore, this should be considered when creating the hd5 file, taking only the proper dates.
-
-Apart from that, the resulting table contains repeated values. From a measurement in a specific period of time, this value is repeated
+The resulting table contains repeated values. From a measurement in a specific period of time, this value is repeated
```
Comment (raimonpv) [inline Electronic-data-warehouse.md:573]: Do you know if this is still true @StevenSong ?

File: Data-on-mad3.md [hunk 2]
```diff
@@ -59,16 +59,11 @@ mkdir ml4c3
 
 Mount to the server:
 ```
-LOGIN=abc123
+LOGIN=$1
```
Comment (erikr) [inline Data-on-mad3.md:62]:
```suggestion
LOGIN=ab123
```

`$1` would get the first argument if called via the shell; perhaps revert to the example with the MGH username?

File: Tensorization.md [hunk 3]
```diff
@@ -19,6 +19,15 @@ Patient lists are defined by an MRN `.csv` file with a column `MRN` and optional
...
 > This process requires mounting our `ml4c3` directory on `mad3`, and the `bedmaster` directory on `PHS-RISC-LM4`.
 
+### ADT table
+ADT stands for Admission Discharge and Transfer. 
+This table contains a time-series of all the movments in the hospital of the patients associated to the cohort. 
+An ADT table lists patients by MRN, their encounter IDs, and the departments they were in. In our pipeline and data analysis, we use ADT tables to define cohorts and identify Bedmaster data.
+
+### xref table
+xref stands for Cross Reference table. This table identifies the Bedmaster files that contains data for each patient in the cohort defined by the ADT table.
+A xref table list patients by MRN, their encounter IDs, and the Bedmaster files associated to their stay with additional metadata.
```
Comment (erikr) [inline Tensorization.md:29]:
```suggestion
The xref table lists patients by MRN, their encounter IDs, the Bedmaster files associated with their stay, as well as additional metadata.
```

**Review by erikr — APPROVED** [review]
Great work! Some minor feedback. Thanks so much.

File: Electronic-data-warehouse.md (hunk 1 above)
Comment (StevenSong) [reply Electronic-data-warehouse.md:573]: this should also be fixed now

**Review by StevenSong — APPROVED** [review]

File: STS-data.md [hunk 4]
```diff
@@ -58,7 +58,7 @@ The `sts-data/outdated-or-raw-format` directory contains the raw STS DB pull dat
 `mgh-bad-ecgs.csv` contains MRNs and acquisition date-times of ECGs that our QC pipeline determined are bad. When we generate bootstraps we exclude these data.
 
 ### Why the 47 selected features in the STS MGH CSV, out of the hundreds more in STS DB?
-The selected features are described in Table 13 of the [STS DB documentation](/docs/sts-db-v2.9.pdf).
+The selected features are described in Table 13 of the [STS DB documentation](/docs/sts-db-v2.90.pdf).
```
Comment (erikr) [inline STS-data.md:61]: Thank you for catching this.

**Review by erikr — APPROVED** [review]

**Pull request merged**